v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-component.tsx

```diff
@@ -93,9 +93,12 @@ export const UnivariateMeasureAdornmentComponent = observer(
     }, [instanceKey, model])
 
     const addLineCoverAndLabel = useCallback((
-      valueObj: IValueObject, labelObj: ILabelObject, adornment: IMeasureInstance
+      valueObj: IValueObject, labelObj: ILabelObject, measure: IMeasureInstance
     ) => {
-      const value = adornment.value
+      const attrId = dataConfig?.primaryAttributeID
+      const value = attrId ? model.measureValue(attrId, cellKey, dataConfig) : undefined
+      if (!value && value !== 0) return
```

Comment:
I'm not sure what we're testing for here -- what's the significance of a falsy value other than `0`?

Reply:
Since `value` could possibly be `undefined` (see line 99), we need to check for that, and if it is undefined and not 0, return before continuing on to render the line. Does that make sense? It does to me but I could be missing something.

Reply:
Yeah, those are better. Thanks 👍